spec/support/vcr_setup.rb

```diff
@@ -0,0 +1,4 @@
+  VCR.configure do |config|
+    config.cassette_library_dir = "spec/tmp/bing/company"
```

Comment:
`tmp` is not a correct dir, `fixtures` is
so it's should be more like `spec/fixtures/external_requests` or similar